packages/components/tests/integration/components/hds/alert/index-test.js

```diff
@@ -172,6 +173,19 @@ module('Integration | Component | hds/alert/index', function (hooks) {
     assert.dom('#test-alert').hasAttribute('aria-labelledby', 'custom-id');
   });
 
+  test('it should pass available a11y automation tests', async function (assert) {
```

Comment:
@MelSumner I'm curious your thoughts about the approach of creating new tests like this or adding `await a11yAudit()` checks into our existing tests that expose more varied iterations of each component's API? Any pros or cons between the two approaches?

Reply:
I think that improves the approach, because it's making sure each iteration passes the available accessibility automation checks. Good idea!

Reply:
follow-up thought: I wonder how much this will slow our tests down if we run this for each iteration? 

As is: 
![CleanShot 2022-08-01 at 14 24 53](https://user-images.githubusercontent.com/4587451/182229544-51de03f2-70d7-4f69-84f6-a4895fe9e25b.png)

Let me try adding the a11y audit to just the different iterations and post those results too.

Reply:
(1992ms vs 5875ms)

Reply:
> @MelSumner I'm curious your thoughts about the approach of creating new tests like this or adding `await a11yAudit()` checks into our existing tests that expose more varied iterations of each component's API? Any pros or cons between the two approaches?

This sounds like a good approach to me but I'm happy with whatever approach you think works best @MelSumner